<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortTest</class>
 <widget class="QDialog" name="SerialPortTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="btnSendClear">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>计数器清零</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopShow">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>停止显示</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearAll">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>清空数据</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveData">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>保存数据</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnManData">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>管理数据</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReturn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="gbSerialSet">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>260</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string>串口参数设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>端口号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxPortName">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>波特率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxBaudRate">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>数据位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxDataBit">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>停止位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxStopBit">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>校验位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboxParity">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpenSerail">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>打开串口</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="txtSendData">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 14pt &quot;宋体&quot;;
color: rgb(255, 255, 127);</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnSendButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送数据</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" rowspan="4">
    <widget class="QGroupBox" name="gdataSet">
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>320</height>
      </size>
     </property>
     <property name="title">
      <string>数据设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="ckHexSend">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>十六进制发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckHexReceive">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>十六进制显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckAutoSend">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自动发送数据</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSpinBox" name="spboxSendTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="ckAutoSave">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自动保存数据</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSpinBox" name="spboxSaveTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cklsDebug">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模拟设备回复数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cklsAutoClear">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>发送数据自动清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QGroupBox" name="gReceive">
     <property name="title">
      <string>接收窗口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTextEdit" name="ReceiveText">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;宋体&quot;;
color: rgb(255, 255, 127);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labReceiveByte">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>接收字节：0字节</string>
      </property>
     </widget>
     <widget class="QLabel" name="labSerialStatus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>串口状态：关闭</string>
      </property>
     </widget>
     <widget class="QLabel" name="labSendByte">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>发送字节：0字节</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
